pkgs/development/python-modules/nsz/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pycryptodome, enlighten, zstandard, kivy
+, withGUI ? true
```

```suggestion
, withGUI ? true
, kivy
```
(the optional dependency should be grouped with the switch)